stable/jenkins/templates/NOTES.txt

```diff
@@ -9,7 +9,7 @@
 
 {{- else if contains "LoadBalancer" .Values.Master.ServiceType }}
 **** NOTE: It may take a few minutes for the LoadBalancer IP to be available.                      ****
-****       You can watch the status of by running 'kubectl get svc -w {{ template "fullname" . }}' ****
+****       You can watch the status of by running 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "fullname" . }}' ****
```

Comment:
didn't notice these were present in the `jenkins chart too. fixed it there now. please take a look.